packages/atomic/src/components/search/atomic-notification/atomic-notification.tsx

```diff
@@ -0,0 +1,85 @@
+import {Component, h, State, Fragment, Prop} from '@stencil/core';
+import {
+  NotifyTrigger,
+  NotifyTriggerState,
+  buildNotifyTrigger,
+} from '@coveo/headless';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {Hidden} from '../../common/hidden';
+import {Bindings} from '../atomic-search-interface/atomic-search-interface';
+import InfoIcon from '../../../images/info.svg';
+import {AriaLiveRegion} from '../../../utils/accessibility-utils';
+import {Heading} from '../../common/heading';
+
+/**
+ * The `atomic-notification` component is responsible for displaying notifications generated by the Coveo Search API (see [Trigger](https://docs.coveo.com/en/1458/tune-relevance/trigger-query-pipeline-feature)).
```

Comment:
```suggestion
 * The `atomic-notification` component is responsible for displaying notifications generated by the Coveo Search API (see [Trigger](https://docs.coveo.com/en/1458)).
```
(everything after the ID is unnecessary)